crates/lgn-api-codegen/src/python/filters.rs

```diff
@@ -46,6 +46,7 @@ pub fn fmt_type(type_: &Type, ctx: &GenerationContext) -> ::askama::Result<Strin
         Type::Array(inner) | Type::HashSet(inner) => {
             format!("list[{}]", fmt_type(inner, ctx).unwrap())
         }
+        Type::Map(inner) => format!("dict[string, {}]", fmt_type(inner, ctx).unwrap()),
```

Comment:
@kirdaybov  We're now supporting hashmaps/dicts in codegen, is this representation in Python ok with you?

Reply:
I think it should be ok, thanks